var/spack/repos/builtin/packages/py-sphinx-argparse/package.py

```diff
@@ -18,4 +18,5 @@ class PySphinxArgparse(PythonPackage):
 
     depends_on('python@2.7.0:2.7,3.5:', type=('build', 'run'))
     depends_on('py-sphinx@1.2.0:', type=('build', 'run'))
+    depends_on('py-poetry', type='build')
```

Comment:
Can you check if this works with `poetry-core` too? @adamjstewart made me aware that poetry people recently split the front-end cli from the core part of the library.

Reply:
Good catch, didn't know about that (I just saw a missing "poetry" error and assumed `py-poetry`), confirmed that it worked.